### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by idkwhatnamele** [GiorgioBrux/nitro-sniper-enhanced#75] [Q&A] [answered]
if I disable all the features such as legitimacyCheck and obfuscationCheck will it make the sniper faster? (heroku)

**Comment by GiorgioBrux** [comment, marked as answer]
Disabling legitimacyCheck, obfuscationCheck, notesCheck will make the sniper faster, but I can't tell by how much.
All the others have no performance impact, and can be leaved on if you wish.

**Comment by idkwhatnamele** [comment]
Alright, but I'm still confused because I didn't snipe anything for almost 2 months so I wonder if the problem is from my sniper or it's just my luck

**Reply by GiorgioBrux** [reply]
Probably just your luck, you can test if the sniper works by going to the heroku console and have someone DM your alt or your main (if you have `useMain`  to true) with a fake discord gift link, like `discord.gift/a`. If something about a fake code shows up, then it's working and the only thing you can do is join more servers/create more alts.

**Comment by idkwhatnamele** [comment]
I have just tested it, but for some reason when I have legitimacyCheck off, a lot of codes don't show up (I tried real/fake codes) and I tested it with my alt and main

**Reply by GiorgioBrux** [reply]
That shouldn't be the case, if you can reproduce the bug please open an issue [here](https://github.com/GiorgioBrux/nitro-sniper-enhanced/issues/new?assignees=&labels=bug&template=bug_report.md&title=%5BBUG%5D).

**Comment by idkwhatnamele** [comment]
Do you suggest leaving legitimacyCheck on true or false?

**Reply by GiorgioBrux** [reply]
I suggest leaving it on `true`, I don't think it has a big performance impact anyway.
